### PR TITLE
Feat: Added PreLoader to Poemify Website

### LINKED ISSUE #20: FEATURE : Adding a PreLoader to the Website
**Is your feature request related to a problem? Please describe.**

The website Doesnt Have a Preloader

**Describe the solution you'd like**

Add a Minimilistic and Elegant Looking PreLoader to the Website that matches the Theme

screencapture-localhost-3000-2024-06-18-21_49_47

**Email**
Enter your email: Shreyash3087@gmail.com

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by shreyash3087**
**Description:**

This PR adds a preloader to the Poemify Website 

**Issue Mentioned:**

Closes #20 

**Video:**

https://github.com/Muhammad-Owais-Warsi/Poemify/assets/72603662/60c04616-8ba6-428b-b7e7-7d6c492ad42d

**Additional Context:**

[X] I am a VSoC Contributor
Email: shreyash3087@gmail.com

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -1,23 +1,75 @@
...
+    <div className="min-h-screen flex flex-col  bg-black">
+      {!completed ? (
+        <div className="flex flex-grow justify-center items-center">
+          {!loading ? (
+            <Lottie
+              options={defaultOptions1}
+              width={"250px"}
+            />
+          ) : (
+            <Lottie
```
Comment (Muhammad-Owais-Warsi) [inline src/app/page.tsx]: please remove this success of the pre loader , the earth symbol alone is sufficient. After the earth symbol is displayed completely ,open up the website. I think returning null or undefined may work in place of <Lottie/> component

**Comment by shreyash3087** [comment]
@Muhammad-Owais-Warsi kindly merge it if theres no issue with the pr

thankyou

**Comment by Muhammad-Owais-Warsi** [comment]
> @Muhammad-Owais-Warsi kindly merge it if theres no issue with the pr
> 
> thankyou

Hey @shreyash3087 please resolve the review , I mean there's a correction to be made. Look onto that

**Comment by shreyash3087** [comment]
@Muhammad-Owais-Warsi I am sorry I am not able to Identify what correction is to be made,it doesnt show any lines that you might have requested changes onto for some reason,can you specify the line and correction that needs to made?

**Comment by Muhammad-Owais-Warsi** [comment]
> @Muhammad-Owais-Warsi I am sorry I am not able to Identify what correction is to be made,it doesnt show any lines that you might have requested changes onto for some reason,can you specify the line and correction that needs to made?

At line nom 53 you have used the Lottie icon (tick mark) for the preloader . If possible can you please remove that

**Comment by shreyash3087** [comment]
@Muhammad-Owais-Warsi Got it,You dont want the tick animation right,i removed it,You can verify.

**Review by Muhammad-Owais-Warsi — APPROVED** [review]

**Comment by Muhammad-Owais-Warsi** [comment]
Hey @shreyash3087 you have used react-lottie, which i have to install on my own machine and then deploy it again , that's not the issue . Issue is when i am installing react-lottie it shows 

> npm warn EBADENGINE Unsupported engine {
npm warn EBADENGINE   package: 'react-lottie@1.2.4',
npm warn EBADENGINE   required: { npm: '^3.0.0' },
npm warn EBADENGINE   current: { node: 'v18.20.3', npm: '10.7.0' }
npm warn EBADENGINE }

I have tried a lot , even i uninstalled the node and re-installed it , chnaged the npm version everything . It works fine, but can't be deployed , is their any other way to resolve this ??

**Comment by Muhammad-Owais-Warsi** [comment]
Hey @shreyash3087 you have used react-lottie, which i have to install on my own machine and then deploy it again , that's not the issue . Issue is when i am installing react-lottie it shows
> 
>  npm warn EBADENGINE Unsupported engine {
>  npm warn EBADENGINE   package: 'react-lottie@1.2.4',
>  npm warn EBADENGINE   required: { npm: '^3.0.0' },
>  npm warn EBADENGINE   current: { node: 'v18.20.3', npm: '10.7.0' }
>  npm warn EBADENGINE }
> 
I have tried a lot , even i uninstalled the node and re-installed it , changed the npm version everything . But the error persists and the site can't be deployed , is their any other way to resolve this ??

**Comment by shreyash3087** [comment]
@Muhammad-Owais-Warsi Don't worry the issue is due to the reason that react-lottie package requires npm v3 but the project is using npm v10, i'll make another pr that will fix this issue by utilizing lottie-react package instead of react-lottie which will be compatible with the current npm version

**Pull request merged**